### PR TITLE
Implemented #5 email notifications

### PR DESCRIPTION
**Pull request by stanislav-sulima**

### CODE REVIEW

**Comment by Ishugaev** [comment]
approved

File: app/Resources/views/mail/post_approved.html.twig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{% extends 'mail/base.html.twig' %}
+
+{% block content %}
+    <p>Hi,</p>
+
+    <p>
+       We glad to inform you that post "{{ post.getTitle }}" has been approved.
```
Comment (holyspecter) [inline app/Resources/views/mail/post_approved.html.twig]: `We are glad...`

Comment (stanislav-sulima) [inline app/Resources/views/mail/post_approved.html.twig]: Stupid mistake. Already fixed

**Comment by holyspecter** [comment]
approved

**Pull request merged**